server.js

```diff
@@ -19,3 +20,14 @@ APP.listen(PORT, function(){
   console.log(PORT);
 }
 );
+
+APP.get('/github/*', proxyGitHub);
```

Comment:
This needs to be above the `listen` command. `listen` starts the server with all the routes above it available, so having this route under it actually does not add it to the server.

Reply:
moved!